### DISCUSSION TITLE
如何降级xray-core

### DISCUSSION THREAD

**Discussion by vincentdxu** [XTLS/Xray-core#171] [Q&A]
问一下，如何可以降级xray-core到一个老一些的版本，最近升级了1.2.1，不太稳定。谢谢

**Comment by RPRX** [comment]
只建议使用最新版，通常包含旧版没有的修复

遇到问题时，建议提具体的 issue 以便解决

**Reply by vincentdxu** [reply]
1.2.1版+passwall在OpenWrt上经常断网。用回1.0版就没问题

**Reply by RPRX** [reply]
> 
> 
> 1.2.1版+passwall在OpenWrt上经常断网。用回1.0版就没问题

这类吗 https://github.com/xiaorouji/openwrt-passwall/issues/879

**Comment by RPRX** [comment]
试试这个 commit https://github.com/XTLS/Xray-core/commit/ae98dc75cf30451d3f29684977ff7324e0fc26e4

**Comment by RPRX** [comment]
试试 https://github.com/xiaorouji/openwrt-passwall/issues/879#issuecomment-763509357
